azavea/pfb-network-connectivity
# Feature/kak/neighborhood pts endpoint#246

## Overview

Adds endpoint to return neighborhood centroids as a GeoJSON feature collection. Stores the centroid on the neighborhood model as a convenience, so it won't need to be recalculated with each request.


### Demo

Example output
```json
{
    "type": "FeatureCollection",
    "features": [
        {
            "id": "8b1ec4ac-cda8-4688-91e0-8ca93902b7e4",
            "type": "Feature",
            "geometry": {
                "type": "Point",
                "coordinates": [
                    -75.16412187001829,
                    39.95259742991459
                ]
            },
            "properties": {
                "name": "philadelphia",
                "label": "Philadelphia",
                "state_abbrev": "PA",
                "organization": "28f2e9b9-03ed-4ead-bcfc-e65e871fdd61"
            }
        }
    ]
}
```


### Notes

As with the main neighborhood endpoint, will restrict results to the neighborhoods in the requesting user's organization.


## Testing Instructions

 * `docker-compose build django`
 * `./scripts/django-manage migrate`
 * `./scripts/server`
 * Check output at http://localhost:9200/api/neighborhood_geojson/


Closes #246
